.github/workflows/linting-frontend.yml

```
@@ -0,0 +1,27 @@
+# https://deno.land/manual@v1.19.2/continuous_integration
+
+name: CI - Linting - Frontend
```

Two dashes are hurting my eyes 👀 😅 
```suggestion
name: CI - Frontend
```